core/Access.php

```diff
@@ -170,18 +171,7 @@ public function reloadAccess(Auth $auth = null)
 
         // case the superUser is logged in
         if ($result->hasSuperUserAccess()) {
-            return $this->reloadAccessSuperUser();
-        }
-
-        // in case multiple calls to API using different tokens, we ensure we reset it as not SU
-        $this->setSuperUserAccess(false);
-
-        // we join with site in case there are rows in access for an idsite that doesn't exist anymore
-        // (backward compatibility ; before we deleted the site without deleting rows in _access table)
-        $accessRaw = $this->getRawSitesWithSomeViewAccess($this->login);
```

Comment:
We do now load those sites only if needed. Before we did this whenever `reloadAccess` was called. Background: Whenever we do `new API\Request()->process()` the access is reloaded and we often did reload the sites without need.